docs/src/js/ds4/main.js

```diff
@@ -13,3 +13,14 @@ $('#banner-menu').on('click', 'a', function(e) {
 $('.expand-btn-example').on('click', function() {
     $(this).attr('aria-expanded', ($(this).attr('aria-expanded') === 'true') ? 'false' : 'true');
 });
+$('.combobox__control').on('click', function() {
```

Comment:
Can we cache `$(this)` as: ```const $this = $('this');```

Reply:
This was quick and dirty. Just as @ianmcburnie mentioned, there's a ton left to improve here, specifically around a11y. I left it out intentionally as I wanted to get this PR through (amongst other work constraints).